api/src/main/java/ai/djl/metric/Metric.java

```diff
@@ -218,6 +253,9 @@ public String toString() {
         if (timestamp != null) {
             sb.append('|').append(timestamp);
         }
+        if (metricType != null) {
```

Comment:
How does this work if you have a metricType but no timestamp?

Reply:
It should work